src/app/Processor/ProcessorDetail/ProcessorDetail.test.tsx

```diff
@@ -0,0 +1,125 @@
+
+    expect(comp.queryByText("Filters")).toBeInTheDocument();
+    expect(
+      comp.queryByText(sinkProcessor.filters?.[0].key as string)
+    ).toBeInTheDocument();
+    expect(comp.queryByText("String contains")).toBeInTheDocument();
+    expect(
+      comp.queryByText(sinkProcessor.filters?.[0].value as string)
+    ).toBeInTheDocument();
+    expect(
+      comp.queryByText(sinkProcessor.filters?.[1].key as string)
+    ).toBeInTheDocument();
+    expect(comp.queryByText("String equals")).toBeInTheDocument();
+    expect(
+      comp.queryByText(sinkProcessor.filters?.[1].value as string)
+    ).toBeInTheDocument();
+
+    expect(
+      comp.queryByText(sinkProcessor.transformationTemplate as string)
+    ).toBeInTheDocument();
+
+    expect(comp.queryByText("Action")).toBeInTheDocument();
+    expect(comp.queryByText("Send to Slack")).toBeInTheDocument();
+    expect(
+      comp.queryByText(sinkProcessor.action.parameters.channel)
+    ).toBeInTheDocument();
+    expect(
+      comp.queryByText(sinkProcessor.action.parameters.webhookUrl)
+    ).toBeInTheDocument();
+
+    expect(comp.queryByText("Source")).not.toBeInTheDocument();
+  });
+
+  it("should display source processor information", async () => {
+    const comp = customRender(<ProcessorDetail processor={sourceProcessor} />);
+    await waitForI18n(comp);
+
+    expect(comp.getByText("Processor type")).toBeInTheDocument();
+
+    expect(comp.getByTestId("processor-type-label")).toHaveTextContent(
+      "Source"
+    );
+
+    expect(comp.queryByText("Filters")).toBeInTheDocument();
+    expect(
+      comp.queryByText(sourceProcessor.filters?.[0].key as string)
+    ).toBeInTheDocument();
+    expect(comp.queryByText("String contains")).toBeInTheDocument();
+    expect(
+      comp.queryByText(sourceProcessor.filters?.[0].value as string)
+    ).toBeInTheDocument();
+
+    expect(
+      comp.queryByText(sourceProcessor.transformationTemplate as string)
+    ).toBeInTheDocument();
+
+    expect(comp.queryByText("Demo source")).toBeInTheDocument();
+    expect(
+      comp.queryByText(sourceProcessor.source.parameters.demoParameter)
+    ).toBeInTheDocument();
+
+    expect(comp.queryByText("Sink")).not.toBeInTheDocument();
+    expect(comp.queryByText("Action")).not.toBeInTheDocument();
+  });
+});
+
+const sinkProcessor: SinkProcessor = {
+  name: "My processor",
+  status: "ready",
+  type: "sink",
+  filters: [
+    {
+      key: "source",
+      type: "stringContains",
+      value: "aws.ec2",
+    },
+    {
+      key: "detail-type",
+      type: "stringEquals",
+      value: "EC2 Instance State-change Notification",
+    },
+  ],
+  transformationTemplate: "Hello, there's a new message: {data.message}",
+  action: {
+    type: "Slack",
+    parameters: {
+      channel: "Demo Channel",
+      webhookUrl:
+        "https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX",
+    },
+  },
+};
+
+const sourceProcessor: SourceProcessor = {
```

Comment:
I'm testing also the different rendering of sources, the source data will be there instead of the action one.

Of course we will have to skip this tests later because we only have sinks at the moment.